tests/cephfs/cephfs_nfs/nfs_multiple_export_using_single_conf.py

```diff
@@ -0,0 +1,115 @@
+        )
+        if not wait_for_process(client=client1, process_name=nfs_name, ispresent=True):
+            raise CommandFailed("Cluster has not been created")
+        out, rc = client1.exec_command(sudo=True, cmd="ceph nfs cluster ls")
+        output = out.split()
+        if nfs_name in output:
+            log.info("ceph nfs cluster created successfully")
+        else:
+            raise CommandFailed("Failed to create nfs cluster")
+        # create nfs
+        export_id1 = secrets.randbelow(1000)
+        export_id2 = secrets.randbelow(1000)
+        psuedo1 = f"/cephfs1_{export_id1}"
+        psuedo2 = f"/cephfs2_{export_id2}"
+        user_id1 = f"nfs.{nfs_name}.{export_id1}"
+        user_id2 = f"nfs.{nfs_name}.{export_id2}"
+        data = [
+            {
+                '"export_id"': export_id1,
+                '"path"': '"/"',
+                '"cluster_id"': f'"{nfs_name}"',
+                '"pseudo"': f'"{psuedo1}"',
+                '"access_type"': '"RW"',
+                '"squash"': '"none"',
+                '"security_label"': "true",
+                '"protocols"': "[4]",
+                '"transports"': '["TCP"]',
+                '"fsal"': {
+                    '"name"': '"CEPH"',
+                    '"user_id"': f'"{user_id1}"',
+                    '"fs_name"': '"cephfs"',
+                },
+                '"clients"': "[]",
+            },
+            {
+                '"export_id"': export_id2,
+                '"path"': '"/"',
+                '"cluster_id"': f'"{nfs_name}"',
+                '"pseudo"': f'"{psuedo2}"',
+                '"access_type"': '"RW"',
+                '"squash"': '"none"',
+                '"security_label"': "true",
+                '"protocols"': "[4]",
+                '"transports"': '["TCP"]',
+                '"fsal"': {
+                    '"name"': '"CEPH"',
+                    '"user_id"': f'"{user_id2}"',
+                    '"fs_name"': '"cephfs_nfs"',
+                },
+                '"clients"': "[]",
+            },
+        ]
+        client1.exec_command(sudo=True, cmd=f"echo {data} > export.json")
+        client1.exec_command(
+            sudo=True, cmd=f"ceph nfs export apply {nfs_name} -i export.json"
+        )
+        out1, rc1 = client1.exec_command(
+            sudo=True, cmd=f"ceph nfs export ls {nfs_name}"
+        )
+        output1 = out1.split()[1:-1]
+        log.info(output1)
+        tf1 = False
+        tf2 = False
+        for ss in output1:
+            if psuedo1 in ss:
+                tf1 = True
+            if psuedo2 in ss:
+                tf2 = True
+            log.info(ss)
+
+        if tf1 and tf2:
+            log.info("export created successfully")
+        else:
+            raise CommandFailed("Failed to export configs")
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
```

Comment:
Do we need to have cleanup for this script

Reply:
added